src/ImageRequest.js

```diff
@@ -31,40 +31,40 @@ class ImageRequest {
 
     this.sharpObject = sharp(this.originalImageBody)
     this.originalMetadata = await this.sharpObject.metadata()
-
-    const qp = this._parseQueryParams()
-
     this.headers = this.event.headers
 
-    if (qp.auto !== undefined) {
```

Comment:
We need to actually check that the value contains `format`. I'm using `this.event.multiValueQueryStringParameters.auto` for that now.